src/admin-views/list.php

```diff
@@ -20,93 +18,108 @@
-				<h4 class="tribe_sectionheader">
-					<?php
-					echo esc_html( apply_filters( 'tribe_events_tickets_module_name', $modules[ $ticket->provider_class ], $ticket->provider_class ) );
-					echo $provider_obj->get_event_reports_link( $post_id );
-					?>
-				</h4>
+		?>
+		<tr class="<?php echo esc_attr( $provider ); ?>" data-ticket-order-id="order_<?php echo esc_attr( $ticket->ID ); ?>" data-ticket-type-id="<?php echo esc_attr( $ticket->ID ); ?>">
+			<!-- (handle, name), price, capacity, available, editlink -->
+			<td class=" column-primary ticket_name <?php echo esc_attr( $provider ); ?>">
+				<span class="ticket_cell_label">Ticket Type:</span>
+				<p><?php echo esc_html( $ticket->name ); ?></p>
+				<button type="button" class="toggle-row"><span class="screen-reader-text">Show more details</span></button>
 			</td>
-		<?php endif; ?>
-		<tr data-ticket-type-id="<?php echo esc_attr( $ticket->ID ); ?>">
-			<td>
-				<p class="ticket_name">
-					<?php
-					printf(
-						"<a href='#' attr-provider='%s' attr-ticket-id='%s' class='ticket_edit'>%s</a>",
-						esc_attr( $ticket->provider_class ),
-						esc_attr( $ticket->ID ),
-						esc_html( $ticket->name )
-					);
-					do_action( 'event_tickets_ticket_list_after_ticket_name', $ticket );
-					?>
-				</p>
 
-				<div class="ticket_controls">
-					<?php echo join( ' | ', $controls ); ?>
-				</div>
+			<?php
+			/**
+			 * Allows for the insertion of additional content into the main ticket admin panel after the tickets listing
+			 *
+			 * @param Post ID
+			 * @since TBD
+			 */
+			do_action( 'tribe_events_tickets_ticket_table_add_tbody_column', $ticket, $provider_obj );
+			?>
 
+			<td class="ticket_capacity">
+				<span class="ticket_cell_label">Capacity:</span>
+				<?php
+				// escaping handled in function
+				echo $ticket->display_original_stock();
+				?>
 			</td>
 
-			<td valign="top">
-				<?php echo $provider_obj->get_price_html( $ticket->ID ); ?>
+			<td class="ticket_available">
+				<span class="ticket_cell_label">Available:</span>
+				<?php
+				if ( 'own' === $ticket->global_stock_mode() ) {
+					echo absint( $ticket->remaining() );
+				} else {
+					echo '(' . absint( $ticket->remaining() ) . ')';
+				}
+				?>
 			</td>
 
-			<td nowrap="nowrap">
-				<?php echo tribe_tickets_get_ticket_stock_message( $ticket ); ?>
+			<td class="ticket_edit">
+				<?php
+				printf(
+					"<button data-provider='%s' data-ticket-id='%s' class='ticket_edit_button'><span class='ticket_edit_text'>%s</span></a>",
+					esc_attr( $ticket->provider_class ),
+					esc_attr( $ticket->ID ),
+					esc_html( $ticket->name )
+				);
+				?>
 			</td>
 		</tr>
 		<?php
-		$count ++;
-	} ?>
+	}
+
+	$modules = Tribe__Tickets__Tickets::modules();
+	?>
+	<thead>
+		<tr class="table-header">
+			<th class="ticket_name">Tickets</th>
+			<?php
+			/**
+			 * Allows for the insertion of additional columns into the ticket table header
+			 *
+			 * @param Post ID
```

Comment:
We should remove this `@param` tag or else pass the post ID in the `do_action()` call